examples/react/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "example-react",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite --port 4000",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "@evervault/react": "workspace:*",
+    "@evervault/ui-components": "workspace:*"
```

Comment:
No they wont. I've added here so that Turbo will run ui-components in dev mode. Might be able to modify example startup script instead and remove this.